nncf/torch/checkpoint_loading.py

```diff
@@ -268,8 +268,26 @@ def run(self) -> Dict[str, torch.Tensor]:
         normalized_keys_to_load = NormalizedKeys(list(self.state_dict_to_load.keys()))
 
         has_legacy_storage_keys = False
+
+        has_version_agnostic_names = False
+        cross_match_key_map = self._cross_match_version_agnostic_names(list(normalized_keys_to_load),
+                                                                       list(normalized_model_keys))
+
+        for matched_checkpoint_key, matched_model_key in cross_match_key_map.items():
+            if matched_checkpoint_key != matched_model_key:
+                has_version_agnostic_names = True
+
+        if has_version_agnostic_names:
+            warnings.warn('Legacy NNCF-enabled .pth checkpoint has been loaded! '
+                          'The version-agnostic `RELU` operator name entries in the state dict have been deprecated. '
+                          'The loader will try to match these entries to the correspoindig `relu` and `relu_` op '
+                          'names. The newly exported checkpoints will be adjusted to the new format.',
+                          category=DeprecationWarning)
+
         for normalized_key_to_load in normalized_keys_to_load:
             key_to_load = normalized_keys_to_load.get_orig_key(normalized_key_to_load)
+            normalized_key_to_load = cross_match_key_map.get(normalized_key_to_load, normalized_key_to_load)
```

Comment:
This function call looks strange. should arguments really be duplicated ?

Reply:
I tried to do the same using an `in` check, and Pylint triggered on it, advising to use `get`. `dict.get` returns the second argument value if the first argument value was not found among the keys of the dict.